native/src/components/TtsPlayer.tsx

```diff
@@ -26,9 +28,11 @@ const StyledTtsPlayer = styled.View<{ $isPlaying: boolean }>`
   align-items: center;
   align-self: center;
   position: absolute;
-  bottom: 5px;
-  min-height: 93px;
+  bottom: 0;
+  min-height: 112px;
   gap: ${props => (props.$isPlaying ? '0px;' : '20px')};
+  margin-bottom: ${props => props.bottom + extraBottomMargin}px;
+  padding: 8px;
```

Comment:
🤔 what do you think of this?
```suggestion
  bottom: ${props => props.bottom + extraBottomMargin}px;
  min-height: 112px;
  gap: ${props => (props.$isPlaying ? '0px;' : '20px')};
  padding: 8px;
```